src/__tests__/lib/containers/CardContainer.test.tsx

```diff
@@ -1,7 +1,7 @@
 import { render, screen } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
 import React from 'react'
-import { SynapseConstants } from '../../../lib'
+import { SynapseConstants } from '../../../lib/utils'
```

Comment:
Changes like this usually resolved the memory leak warning on a test suite...not sure why